lib/trackler/problem.rb

```diff
@@ -10,78 +10,99 @@ def initialize(slug, root)
     end
 
     def exists?
-      !!md && !!yaml
+      !!description && !!metadata
```

Comment:
Of the three options:

1. `description && metadata`
2. `!!description && !!metadata`
3. ` !!(description && metadata)`

I prefer 2 for both readability and correctness.